plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -3663,8 +3669,12 @@ public static void cleanExtraOutputFiles(HdfsEnvironment hdfsEnvironment, HdfsCo
             Iterator<String> filesToDeleteIterator = filesToDelete.iterator();
             while (filesToDeleteIterator.hasNext()) {
                 String fileName = filesToDeleteIterator.next();
-                log.debug("Deleting failed attempt file %s/%s for query %s", path, fileName, queryId);
-                checkedDelete(fileSystem, new Path(path, fileName), false);
+                String filePath = path + "/" + fileName;
```

Comment:
You could use `new Path(path, fileName)` here, so you have one variable, not two